### ISSUE TITLE
[Modularization] Move pyopenms into own repository

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6700]
To have a separate release process, versioning etc.
It might be a bit confusing for users at first but I think this will make the release much more manageable.

**Comment by timosachsenberg** [comment]
and include it as submodule?

**Comment by jpfeuffer** [comment]
Not really. Can be completely separate in my opinion. We would then in the pyopenms repo just install openms-lib with conda and build on top.

**Comment by timosachsenberg** [comment]
I think this does currently not work well with our release process / ci . should we still pursue this or is a monorepo style here more appropriate?
